airflow/providers/google/cloud/hooks/vertex_ai/endpoint_service.py

```diff
@@ -91,6 +92,7 @@ def create_endpoint(
         :param project_id: Required. The ID of the Google Cloud project that the service belongs to.
         :param region: Required. The ID of the Google Cloud region that the service belongs to.
         :param endpoint: Required. The Endpoint to create.
+        :param endpoint_id: The ID of Endpoint. If not provided, Vertex AI will generate a value for this ID.
```

Comment:
Maybe we should add info about the format of this param?

Reply:
Agree. Add the format by https://github.com/apache/airflow/pull/23070/commits/cc985e22e971454f33906006b57d699e4b1ad931. Thanks!

Reply:
Also updated with valid `TEST_ENDPOINT_ID` value by https://github.com/apache/airflow/pull/23070/commits/58dfa78f1dadf66e6095eff852b5a3557103e88a